### ISSUE TITLE
Bug: Official Government Banner

### ISSUE THREAD

**Issue by anna-zhang** [GSA/digitalgov.gov#3027]
**Describe the bug**
When viewing https://digital.gov/ on a mobile device, the icon (X) to close the official government banner does not show up. 

**Steps to Reproduce**
1. Go to https://digital.gov/ on a mobile device
2. Click on the official government banner at the top of the page
3. See error in the top right corner (missing the close icon)

**Screenshots**
![image](https://user-images.githubusercontent.com/31214438/87798366-ea5e2f00-c819-11ea-985f-60658b485948.png)

**Comment by danielimmke** [comment]
I'd like to claim this!
